app/lib/fsHelper.js

```diff
@@ -18,7 +28,7 @@ function saveJsonSync(obj, filename) {
 
 function loadJsonSync(filename) {
   const path = `${OUTPUT_DIR}/${filename}.json`;
-  const jsonString = fs.existsSync(path) ? fs.readFileSync(path, 'utf8') : undefined;
+  const jsonString = fileExists(path) ? fs.readFileSync(path, 'utf8') : undefined;
```

Comment:
It seems you have interpreted the changes in the same way. The reason I've highlighted this line is because I'd argue it does not adhere to the recommended approach of trying to use the file rather than checking for its' existence.
Specifically, if `fileExists` passes it then goes on to read the file.